config/environments/production.rb

```diff
@@ -71,11 +70,11 @@
   # config.action_mailer.raise_delivery_errors = false
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
-  # English when a translation cannot be found).
-  config.i18n.fallbacks = [:en]
+  # the I18n.default_locale when a translation cannot be found).
+  config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
New config here is to use this setting in production, and keep using the old style in dev/test.